### DISCUSSION TITLE
Signature input/drawing field?

### DISCUSSION THREAD

**Discussion by brandonhon** [Budibase/budibase#3557] [Ideas]
Is there a way to add a signature box? I'll be using my app/form on an iPad and need the ability to take a signature.

**Comment by Fabian1410** [comment]
I think its not possible right now. You need to save the "image" signature in the DB manual.

**Reply by mjashanks** [reply]
Correct :)

We would need to add a new component for this.

**Reply by tazilios** [reply]
Is this feature somewhere in your backlog? Any plan?

**Comment by Winteriver** [comment]
Signature field would be highly appreciated, as "underwriting" on mobile devices is still the way to go.
No one has his/her digital signature available, if using Budibase on a mobile device.

A (separate?) drawing area could be really helpful to make small notes/drawings on a mobile device to make a situation "more understandable" or to visually describe a situation - especially since even the cheapest iPads today have the option for a digital pencil...

**Comment by michaeldeborst** [comment]
Bump, Other tools like Frappe Framework do have space for this but are not as easily combined with extra systems like Budibase offers. A signature field would be great as a way to incorporate budibase in other workflows regarding corporate approvals and signatures.

**Comment by MrApe** [comment]
+1 for this!!

**Comment by rafikApp** [comment]
+1 for this
